core/pom.xml

```diff
@@ -173,14 +173,15 @@
       <artifactId>xalan</artifactId>
       <version>${xalan.version}</version>
       <scope>provided</scope>
-    </dependency>
-    <dependency>
-      <groupId>xerces</groupId>
-      <artifactId>xercesImpl</artifactId>
-      <version>${xercesImpl.version}</version>
-      <scope>provided</scope>
-    </dependency>
-    <dependency>
+   </dependency>
```

Comment:
Looks like the indentation here is not correct. At least not the same as the rest of the file.